### ISSUE TITLE
Add support for Notebook tab widgets

### ISSUE THREAD

**Issue by sanpii** [antoyo/relm#83]
Hello,

Like #31 but for tab widgets, something like that:

```rust
view! {
    gtk::Notebook {
        gtk::Button {
            tab: {
                tab_widget: LabelWidget(model_param),
            }
        }
    }
}
```

**Comment by antoyo** [comment]
Here's my idea to fix this issue:
> [I just thought that I could change relm so that we could specify the required word (like cell) instead of packing. This seems like a better idea as it does not make any assumption. I'll try to do that soon, but since I'm really busy these days, it will probably not be done until next year.](https://gitter.im/relm-rs/Lobby?at=5a09a31a026aeaa02f0a4e2a)

**Comment by antoyo** [comment]
I fixed the issue in commit 35c0f28a7647c4637fe33763f0f0a00aec3d54e0 and you can see [an example here](https://github.com/antoyo/relm/blob/master/examples/tabs.rs#L70-L72).
These changes are not published yet and you will need to construct the widget differently if it is a relm widget.
